product_docs/docs/pgd/5/manual/07-configure-proxies.mdx

```diff
@@ -0,0 +1,247 @@
+---
+title: Step 7 - Configure proxies
+navTitle: Configure proxies
+deepToC: true
+---
+
+## Configure proxies
+
```

Comment:
How about adding some context here - what purpose does the Proxy serve? Why is it useful?

Reply:
Added:

PGD can use proxies to direct traffic to one of the clusters nodes, selected automatically by the cluster. There are performance and availabilty reasons for using a proxy:

    Performance: By directing all traffic and in particular write traffic, to one node, the node can resolve write conflicts locally and more efficiently.
    Availability: When a node is taken down for maintenance or goes offline for other reasons, the proxy can automatically direct new traffic to a new, automatically selected, write leader.

It is best practice to configure PGD Proxy for clusters to enable this behavior.